### ISSUE TITLE
No fast forward in the sdl2 video driver

### ISSUE THREAD

**Issue by hyarsan** [libretro/RetroArch#5628]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

No fast forward in the sdl2 video driver when it should be there.
There is working slow-motion though.

### Expected behavior

Fast forward in the sdl2 video driver

### Actual behavior

No fast forward in the sdl2 video driver

### Steps to reproduce the bug

1. Turn on the sdl2 video driver and turn on an compatible menu driver ( rgui )
2. Try to fastforward like you normally would

### Bisect Results

This was always the case.

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.6.7

### Environment information

- OS: Windows 10

**Comment by orbea** [comment]
Is there any reason you are using the SDL driver? I understand its mostly there as a last resort and the other drivers should be much better supported.

**Comment by hyarsan** [comment]
Yes, it runs alot better than Vulkan or OpenGL. The others seem to also work okay, but SDL doesn't have weird resize issues for one.

**Comment by orbea** [comment]
Well that is unfortunate, I wish I could offer something more helpful. :( The SDL driver has been neglected and is missing features like this because its often inferior to the other video drivers, but I guess that is not always true...

**Comment by hyarsan** [comment]
Well luckily on this PC it isn't more or less the case but on my other two laptops, I'm pretty sure it still is.

**Comment by hyarsan** [comment]
It turns out that turning off Vsync works.

**Comment by orbea** [comment]
@hyarsan I can not reproduce this with `SDL-2.0.9`, fast forward works with and without vsync. Can you reproduce this still?
